spec/services/next_lesson_spec.rb

```diff
@@ -10,15 +10,15 @@
 
   describe '#to_complete' do
     context 'when there are incomplete lessons after the most recently completed lesson' do
-      let(:lesson_completions) { [create(:lesson_completion, lesson: lesson_one)] }
+      let(:lesson_completions) { Array.new(1) { create(:lesson_completion, lesson: lesson_one) } }
```

Comment:
This was a slightly odd one. One cop wanted it to be `1.times.map`, then the performance cop triggered to want this instead.

Reply:
Nit Would using a Factorybot list here make this easier to figure out whats going on / read at a glance?
 
```suggestion
      let(:lesson_completions) { create_list(:lesson_completion, 1, lesson: lesson_one) }
```

Or maybe we could make it the services responsibility to always make sure lesson completions are a collection in the initializer?

```ruby
@lesson_completions = Array(lesson_completions)
```

Whats your thoughts?

Reply:
I like the readability / semantics of `create_list`, updated 👌. 

Don't mind the casting in the service either, but I'm going to go with the first option to minimise churn amongst all the other small changes